Packs/Palo_Alto_Networks_Enterprise_DLP/ReleaseNotes/2_0_5.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### DLP - Get User Feedback
+
+Added the `Exception requested` feedback to the playbook flow, after the user requests for an exemption. 
```

Comment:
Myabe worth to mention that this will update the incident on PANDLP?

Reply:
Sure, will add it.